nilearn/plotting/displays.py

```diff
@@ -184,6 +185,23 @@ def draw_position(self, size, bg_color, **kwargs):
                 **kwargs)
 
 
+def _coords_3d_to_2d(coords_3d, direction):
+    """Project 3d coordinates into 2d ones given the direction of a cut
+    """
+    direction_to_index = {'x': slice(1, None),
```

Comment:
Is there a reason to use slice here? Using integer values for axis seem more natural to me.